app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -58,7 +58,7 @@
 import kotlin.Lazy;
 import timber.log.Timber;
 
-public class PlaybackController {
+public class PlaybackController implements PlaybackControllerNotifiable {
```

Comment:
> I'm not super happy with PlaybackControllerNotifiable

You could use the same nomenclature as SessionRepository. Interface: `SessionRepository`, Class: `SessionRepositoryImpl`

Reply:
In this case the `Impl` class provides the implementation whereas our interface is an observer, so what do we call the equivalent of `SessionRepository` in your example?

`PlaybackControllerRepository` doesn't really fit as its not a MVVM component :thinking: